coop_shift/model/shift_leave.py

```diff
@@ -123,12 +123,6 @@ def _check_partner_leaves(self):
                             other_leave.stop_date and other_leave.stop_date or
                             _('Undefined')))
 
-    # Overload Section
-    @api.multi
-    def copy(self, default=None):
-        raise ValidationError(_(
-            "You can not duplicate a leave : Unimplemented Feature"))
-
```

Comment:
Why was this removed?

I'm not sure of the functional implication of removing these lines. I'd prefer to keep them, just in case

Reply:
@ivantodorovich : this is just blocking duplicate leave but I don't see any functional use of it

Reply:
Can you please put it back?
We don't want to change any functional behaviour. Also Simon just confirmed this was intentional